build.gradle

```diff
@@ -386,6 +387,20 @@ gradle.projectsEvaluated {
   }
 }
 
+// test retry configuration
+boolean isCiServer = System.getenv().containsKey("CI")
+subprojects {
+  apply plugin: "org.gradle.test-retry"
+  tasks.withType(Test).configureEach {
+    retry {
+      if (isCiServer) {
```

Comment:
You could probably just not configure the task if it is not CI:

```
if (isCiServer) {
tasks.withType(Test).configureEach {
    retry {
     ....
    }
}
```

Reply:
Also, I would highly recommend to set `maxFailures` to some reasonable value, `5` for example:

```
     * The maximum number of test failures that are allowed before retrying is disabled.
```

Wdyt?

Reply:
We would really benefit from something like https://github.com/gradle/test-retry-gradle-plugin/issues/29 for our use case to catch the worst offenders.
I agree with giving it a kick start with 1-2 retries and stopping on multiple test failures. From what I have noticed, it usually takes 2 retries recently to get it going. But let's start small enough to give a boost and not over rely.

I'll update the PR with `{ maxFailures = 10, maxRetries = 1 }`.